frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -163,6 +212,40 @@ class MeshesView extends React.Component<Props, State> {
       Store.dispatch(setPositionAction(seedPosition, null, false));
     };
 
+    const startComputingMeshfile = async () => {
+      await startComputeMeshFileJob(
+        this.props.organization,
+        this.props.datasetName,
+        this.props.segmentationLayer.fallbackLayer || this.props.segmentationLayer.name,
+        this.props.activeResolution,
+      );
+      this.setState({ isComputingMeshfile: true });
+      Toast.success(
+        "The computation of a mesh file was started successfully. Depending on the dataset's size this may take a bit. You don't need to keep this dataset open for the computation to continue.",
+      );
+    };
+
+    const getJobsDisabledTooltip = node => (
+      <Tooltip
```

Comment:
Won't this always show the tooltip (even without hovering) if jobs are disabled? I think, you want something like this:

```js
      <Tooltip
        title={features().jobsEnabled ? "Compute a mesh file for this dataset so that meshes for segments can be loaded quickly afterwards." : "The computation of mesh files is not supported by this webKnossos instance."}
      >
```

Reply:
Oh, yes. I don't know how this slipped by without me noticing. Thanks for pointing it out!